packages/fyndiq-component-alert/src/index.js

```diff
@@ -57,34 +60,39 @@ export default class Alert extends React.Component {
   }
 
   render() {
-    const { children, type, unclosable } = this.props
+    if (this.state.count < 5) {
+      console.log(this.state.count)
+      saveState(this.state.count + 1)
```

Comment:
So I think the behavior should rather be "if the user closes the popup 5 times we should stop showing it" rather than having the `render` function incrementing the number. To do so, you can use the `close()` method.